doc/services/pm/system.rst

```diff
@@ -3,42 +3,51 @@
-on the selected power management policy and the duration of the idle time
-allotted by the kernel.
-
-It is an application responsibility to set up a wake up event. A wake up event
-will typically be an interrupt triggered by one of the SoC peripheral modules
-such as a SysTick, RTC, counter, or GPIO. Depending on the power mode entered,
-only some SoC peripheral modules may be active and can be used as a wake up
-source.
+The kernel enters the idle state when it has nothing to schedule.
+Enabling :kconfig:option:`CONFIG_PM` allows the kernel to call upon the
+power management subsystem to put an idle system into one of the supported power states.
+
+The PM subsystem selects the appropriate power state to transition to
+depending on the following factors:
+
+* The amount of time prescribed by the kernel to suspend upon invocation of the power manager.
+* The chosen power management policy.
+
+It is the application's responsibility to set up a wake-up event.
+A wake-up event will typically be an interrupt triggered by an SoC peripheral module.
+Examples include a SysTick, RTC, counter, or GPIO.
+Keep in mind that depending on the SoC and the power mode in question,
+not all peripherals may be active, and therefore
+some wake-up sources may not be usable in all power modes.
 
 The following diagram describes system power management:
 
 .. image:: images/system-pm.svg
    :align: center
    :alt: System power management
 
-Some handful examples using different power management features:
-
-* :zephyr_file:`samples/boards/stm32/power_mgmt/blinky/`
-* :zephyr_file:`samples/boards/esp32/deep_sleep/`
-* :zephyr_file:`samples/subsys/pm/device_pm/`
-* :zephyr_file:`tests/subsys/pm/power_mgmt/`
-* :zephyr_file:`tests/subsys/pm/power_mgmt_soc/`
-* :zephyr_file:`tests/subsys/pm/power_states_api/`
 
 Power States
 ============
 
-The power management subsystem contains a set of states based on
-power consumption and context retention.
+The power management subsystem defines a set of states described by the
+power consumption and context retention associated with each of them.
+
+The high level description of each state is as follows, each having more or less
+the cumulative effects of the previous:
 
-The list of available power states is defined by :c:enum:`pm_state`. In
-general power states with higher indexes will offer greater power savings and
-have higher wake latencies.
+* ACTIVE: System is fully powered and active.
+* RUNTIME_IDLE: Cores enter deepest idle state waiting for interrupts,
+  all other devices are unaffected.
+* SUSPEND_TO_IDLE: May put some peripherals into low power states.
+* STANDBY: Peripherals are put into low power states, and non boot cores are powered off.
+* SUSPEND_TO_RAM: Powers off as much of the system as possible.
+  Execution state and device state is held in ram.
+* SUSPEND_TO_DISK: saves to a disk instead of ram.
+* SOFT_OFF: essentially nothing is preserved, wake-up will be like a system reboot.
```

Comment:
I am not a big fan of duplicate this information. Can we just link to their definition ?

Reply:
Fair enough